docs/release-notes/v11.1.0.md

```diff
@@ -0,0 +1,139 @@
+---
+id: v11.1.0
+title: Version v11.1.0 Release Notes
+sidebar_label: v11.1.0
+image: "img/release-note-link-preview.png"
+---
+
+_May 25th, 2023_
+
+## Console
+
+### New Features
+
+#### User Groups Management
+
+From a dedicated section inside of the Company Overview, Company Owners can now manage Company users also through Groups.  
+Thanks to Groups it is possible to simplify the management of a multitude of users who have to be assigned the same privileges (e.g. a work team).  
+Management of users through groups sits alongside the actual individual management of users, to allow you to better structure access to resources as you prefer, giving you the freedom to choose how to govern access levels of your Company users.  
+
+To discover more about user groups management, just take a look at the related [documentation page](../development_suite/identity-and-access-management/group-management.md). 
+
+# Privatized Marketplace for Companies
+
+From now on, Companies can have their own private Marketplace items, which will not be visible and usable from those who are not part of those specific Companies.   
+More specifically, Company Owners and Project Administrators can now be able to manage – directly from CMS – Marketplace resources that will be available exclusively for the members of their Company. This has to be done 
+by specifying the `tenantId` as a Company identifier. 
+
+To discover more information, visit the dedicated Marketplace [documentation page](../marketplace/add_to_marketplace/contributing_overview#public-and-private-marketplace.md).
+
+### Improvements
+
+### Bug Fix
+
+#### Error displaying Debug section
+
+A bug that caused the Debug Area of the Console to fail to load has been fixed.
+
+#### CI/CD tool malfunction in project creation
+
+It has been fixed a bug in the Project creation that caused a malfunction in the display of the default CI/CD tool of each environment.
+
+#### "Undefined" in the detail of a Pod error
+
+We fixed an issue that prevented the user to understand the type of error running in a Pod, by showing a generic "undefined" in the information popup.
+
+## Fast Data
+
+### New Features
+
+#### Ingestion Reloader
+
+With the new version of the Ingestion Reloader, the `1.2.0`, you can now filter specific messages retrieved from a bucket, using a custom selector system and allowing precise and targeted filtering of messages to be reingested: you just need to specify the file with it in the `customSelectorName` field. The selector's configurations are available at the POST `/reingestion/file` and POST `/reingestion/topic` endpoints.
+
+#### Debezium DB2 plugin
+You can now find in the Marketplace the [Debezium connector plugin for DB2](/documentation/docs/fast_data/connectors/debezium_cdc#db2.md) databases.
```

Comment:
@giuliowaitforitdavide let's try to write similar stuff of the past release notes

Reply:
Done, let me know if it's ok